emulators.tex

```diff
@@ -8,14 +8,127 @@ \chapter{Emulators}
 
 Download the MEGA65 emulator source code from \url{https://github.com/lgblgblgb/xemu}.
 
-Download pre-compiled versions from \url{https://github.lgb.hu/xemu/}.
+Download pre-compiled versions from \url{https://github.lgb.hu/xemu/}. Installers are
+available for Apple Macintosh, Windows and Linux.
 
-A live ISO image containing the emulator, documentation, and other tools is available from \url{Forum64.de}
+A live ISO image containing the emulator, documentation, and other tools is available from
+Forum64.de
 at \url{https://www.forum64.de/index.php?thread/104698-xemu-live-system-iso-file/\&postID=1549927\#post1549936}.
+Should you choose to use it you may find instructions on page \pageref{sec:live-iso-image}.
 
 \section{Using The Xmega65 Emulator}
+\index{emulator}
+
+\subsection{The keyboard}
+\index{keyboard}
+
+When using a real Mega 65 a keyboard layout related to the old Commodore machines is
+encountered. How can this be adapted to a PC/Mac ? It's not that hard. Most of the keys
+for writing stay the same - given an American layout. So also special characters reached
+by \specialkey{Shift} and a number are same to a U.S. keyboard.
+
+The closer you get to \specialkey{Enter} or \widekey{Backspace} keys you will find
+a whole column of keys missing. Yes, even a C64-keyboard had this extra set of keys. For
+Xemu those are mainly operated with the six keys above the cursor keys. There you also
+find \specialkey{RUN STOP}, completely off in comparison to C64 emulator like VICE. The reason is, a Mega 65 even has an \specialkey{Esc} key. So some keys are approaching a
+more modern layout.
+
+\begin{center}
+  \includegraphics[width=\linewidth]{images/xemu-extended-keyboard.png}
+\end{center}
+
+As you might spot quickly with this setup \specialkey{RUN STOP} \widekey{RESTORE} could
+be done single-handed.
+
+Please be aware this image shows graphics symbols being reached in the orientation the
```

Comment:
How about:

"This diagram shows the PETSCII glyphs as they appear on a MEGA65 keyboard, with the \megasymbolkey glyph on the left and the \specialkey{Shift} glyph on the right. Note that on a PC keyboard, the key for \megasymbolkey is to the right of the \specialkey{Shift}."

I don't think the example is necessary to explain.

Reply:
I don't have an English mother's tongue. So your condensed version will be much simpler to be understood.